README.md

```diff
@@ -41,106 +41,67 @@
-
-- 🌐 **Deploy a centralized BeeAI instance** that the entire team can access
-- 📚 Create a **team catalog** where developers publish and end users discover agents
-- 🧰 **Standardize agent interfaces** for consistent user experiences
-- 🔐 **Centrally manage** LLM connections to control costs and access
+BeeAI is an open-source platform that makes it easy to test, debug, and share AI agents in an interactive UI with out-of-the-box trajectory, citations, and more. Built on the [Agent2Agent (A2A) Protocol](https://a2a-protocol.org/) and hosted by the Linux Foundation, BeeAI bridges the gap between different agent ecosystems.
 
 ## Key Features
 
-| Feature | How It Works | Business Value |
-| :------ | :----------- | :------------- |
-| **Agent Catalog** | One BeeAI platform serves your entire team | Everyone works from the same system with unified management |
-| **Framework Agnostic** | BeeAI implements the Agent2Agent (A2A) Protocol to standardize agent interfaces regardless of how they're built | Developers use their preferred tools while maintaining compatibility |
-| **Containerized Agents** | Each agent runs in its own container with defined resource limits | Better performance, improved security, and efficient resource usage |
-| **Consistent Interfaces** | Predictable agent interactions | Learn once, use everywhere |
-| **Agent Discovery** | All agents appear in a searchable catalog with capability details | End users easily find agents and developers see usage patterns |
-| **LLM Provider Flexibility** | Connect to any LLM provider | Use the best model for each task and easily switch providers |
+| Feature | Description |
+|:---------|:-------------|
+| 🎯 Instant Agent UI | Generate a shareable front-end from your code in minutes. Focus on your agent's logic, not UI frameworks. |
+| 🚀 Effortless Deployment | Go from container to production-ready. We handle database, storage, scaling, and RAG so you can focus on your agent. |
+| 🔄 Multi-Provider Playground | Test across OpenAI, Anthropic, Gemini, IBM watsonx, Ollama and more. Instantly compare performance and cost to find the optimal model. |
+| 🔧 Framework-Agnostic | Run agents from LangChain, CrewAI, BeeAI and more on a single platform. Enable cross-framework collaboration without rewriting your code. |
 
 ## Quickstart
 
-> [!TIP]
-> This is the short version. See the [installation guide](https://docs.beeai.dev/introduction/quickstart) for detailed instructions.
-
-1. **Install** the BeeAI CLI using [uv](https://docs.astral.sh/uv/) (on Linux you'll also need [QEMU](https://www.qemu.org/download/#linux)):
+### Installation
 
 ```sh
-uv tool install beeai-cli
+sh -c "$(curl -LsSf https://raw.githubusercontent.com/i-am-bee/beeai-platform/HEAD/install.sh)"
 ```
 
-2. **Start** the BeeAI platform:
+This interactive script will automatically:
+- Install prerequisites (QEMU on Linux)
+- Start the BeeAI platform
+- Configure your LLM provider
+- Open the web interface
```

Comment:
I think this part is not strictly necessary to know in order to start, we can drop it to make the instructions shorter